Latitude/index.html

```diff
@@ -0,0 +1,244 @@
+                <article class="description">
+                    <h3>Dapibus etiam lorem</h3>
+                    <p>Phasellus dapibus sed convallis feugiat ipsum. Donec tesque turpis sed amet ultrices vestibulum
+                        nulla.</p>
+                    <a href="#" class="btn btn-one">Learn More</a>
+                </article>
+            </section>
+
+            <section class="image-card">
+                <img src="./assets/pic03.jpg" alt="library" class="images">
+                <article class="description">
+                    <h3>Sed imperdiet adipiscing</h3>
+                    <p>Phasellus dapibus sed convallis feugiat ipsum. Donec tesque turpis sed amet ultrices vestibulum
+                        nulla.</p>
+                    <a href="#" class="btn btn-one">Learn More</a>
+                </article>
+            </section>
+            <hr>
+        </article>
+
+        <article class="description-card desktop-card">
+            <section class="large-image">
+                <img src="./assets/pic04.jpg" alt="Street" class="images">
+                <article class="description description-one">
+                    <h3>Feugiat phasellus sed consequat</h3>
+                    <p id="para-one">Lorem ipsum dolor sit amet veroeros</p>
+                    <p>Phasellus dapibus convallis scelerisque. Donec tempus augue id tortor ultricies eget pellentesque
+                        turpis ultrices. Vestibulum ut nulla sem, vel iaculis arcu. Nulla vel lorem nisl, quis ultricies
+                        tellus. Nunc imperdiet elit sed dolore lacus venenatis rutrum. Vivamus vulputate urna dolor est
+                        sagittis purus, in laoreet.</p>
+                    <a href="#" class="btn btn-one">Learn More</a>
+                </article>
+            </section>
+            <section class="large-image">
+                <img src="./assets/pic05.jpg" alt="Forest" class="images">
+                <article class="description description-one">
+                    <h3>Dapibus lorem luctus volutpat rutrum</h3>
+                    <p id="para-one">Lorem ipsum dolor sit amet veroeros</p>
+                    <p>Phasellus dapibus convallis scelerisque. Donec tempus augue id tortor ultricies eget pellentesque
+                        turpis ultrices. Vestibulum ut nulla sem, vel iaculis arcu. Nulla vel lorem nisl, quis ultricies
+                        tellus. Nunc imperdiet elit sed dolore lacus venenatis rutrum. Vivamus vulputate urna dolor est
+                        sagittis purus, in laoreet.</p>
+                    <a href="#" class="btn btn-one">Learn More</a>
+                </article>
+            </section>
+            <hr>
+        </article>
+        <article class="large-view-image">
+            <article class="description-card description-two">
+                <h3>Dolore rutrum arcu nullam ultricies</h3>
+                <p id="para-one">Lorem ipsum dolor sit amet veroeros</p>
```

Comment:
sug: just use para don't use like one two three

Reply:
so becaruse Iam already use that  para keyword, that's why I taken  para-one  but next I  time will taken better way as you suggested